Added two components with respective Test pages. BookNow is only a button and payment is a form with the summary.

Surafelgit opened the pull request:
they are test pages that will be integrated with other pages when needed. accessed on /book and /pay url.

Surafelgit committed:
Added two components with respective Test pages

they are test pages that will be integrated with other pages when needed. accessed on /book and /pay.

TrustfulDev committed:
Adds the book now button to the details page

It routes to the payment page and the details page still has filler information. Backend needs to work on getting the hotel database out.

TrustfulDev commented:
I removed the test page for the Book Now button --> Moved it to the Details Page where it belongs. 
- The button routes to the payment page
- Details Page is still filler data (NEED BACKEND TO FINISH hotel db)

TrustfulDev committed:
Fixed the css for the payment component

There is a test page being used for the payment component. This needs to be deleted later down the road.

TrustfulDev reviewed (approved):
Looks good. Going to approve so Albany can work with it.

reddesignsguy reviewed (approved):
gud
